docs/src/pages.js

```diff
@@ -28,6 +28,7 @@ datagrid.children =
         { pathname: '/components/data-grid/getting-started' },
         { pathname: '/components/data-grid/columns' },
         { pathname: '/components/data-grid/rows' },
+        { pathname: '/components/data-grid/sorting', title: 'Sorting' },
         { pathname: '/components/data-grid/filtering', title: 'Filtering' },
```

Comment:
Filtering is more popular #925. Why include a `title`?

```suggestion
        { pathname: '/components/data-grid/filtering' },
        { pathname: '/components/data-grid/sorting' },
```

x2